data/monitoring/prom-rule.yaml

```diff
@@ -37,3 +37,15 @@ spec:
           for: 5m
           labels:
             severity: warning
+        - expr: sum(up{namespace='{{ .Namespace }}', pod=~'kubemacpool-mac-controller-manager-.*'} or vector(0))
+          record: kubevirt_cnao_kubemacpool_manager_num_up_pods_total
+        - expr: sum(kubevirt_cnao_cr_kubemacpool_deployed{namespace='{{ .Namespace }}'} or vector(0))
```

Comment:
Why do we need the sum here? Can't we just remove true or false from kubevirt_cnao_cr_kubemacpool_deployed ? We do not care about the number of deployed objects here, do we? Only about whether it is enabled or not

Reply:
the alert is using these 2 records `if(x==1 && y==0) for >5min then fire!` 
In order for the alert to work, the records needs to point to the same entry (x counts apples (arbitrary gauge) and y counts bananas (num of up kmp pods) )
After some playing with this with @sradco, we saw that `sum()` normalizes the records to be of same in "type" and so it works.